### DISCUSSION TITLE
images/connectors.gif

### DISCUSSION THREAD

**Discussion by macmarrum** [freeplane/docs#24] [General]
@ofossum thanks for creating the animated `images/connectors.gif` on https://freeplane.github.io/wiki/#/understanding-nodes?id=connectors

I noticed that there is a black stripe at the bottom, with characters/words appearing on it. I could see it reflected the node text that was being typed it, but I was confused because not all of the words made it into the nodes. Then I realized they were the description of keyboard keys in German (einfg, Strg).

In general, captions might be quite useful to beginners 👍 Yet, the documentation is in English, and I am concerned that German captions can bring more confusion than benefit.

What do you think?

**Comment by ofossum** [comment]
Hi and sorry for my delayed answers!
I just started a new job and had much less time than i anticipated beforehand. ;)

Yes, i totally agree, that german captions were kind of bad. 
It was more like a test, how this works and how it would look on the page and i hadn't found out, where to localize this caption-thingy.
I'd try again to do something similar and see if i can have those captions in english for other images, where an animated demonstration comes in handy.

**Reply by macmarrum** [reply]
Cool 👌 
And good luck in your new job 😀

**Reply by ofossum** [reply]
Thank you! ;)
